Subject: testdriver: Add portgroups example to test-many-devices
From: Cole Robinson crobinso@redhat.com Sat Mar 2 10:25:35 2024 -0500
Date: Sat Mar 2 13:22:40 2024 -0500:
Git: cf172c8e575ad3ee026c29f6e9f6222f524f0fe8

This helps hit some uitests code coverage

Signed-off-by: Cole Robinson <crobinso@redhat.com>

diff --git a/tests/data/testdriver/testdriver.xml b/tests/data/testdriver/testdriver.xml
index 4e0a41ed..58af8129 100644
--- a/tests/data/testdriver/testdriver.xml
+++ b/tests/data/testdriver/testdriver.xml
@@ -355,7 +355,10 @@ Foo bar baz &amp; yeah boii &lt; &gt; yeahfoo
       <source network='dupmac'/>
       <mac address='22:22:33:54:32:11'/>
     </interface>
-    <interface type='user'/>
+    <interface type='network'>
+      <source network='plainbridge-portgroups' portgroup='sales'/>
+      <mac address='22:22:00:54:32:11'/>
+    </interface>
     <interface type='bridge'>
       <source bridge='brempty'/>
       <mac address='22:22:33:44:AA:BB'/>
